agent/internal/containers.go

```diff
@@ -449,6 +449,7 @@ func (c *containerManager) reattachContainer(
 		}
 		return nil, errors.New(errorMsg)
 	}
+	ctx.Ask(containerRef, actor.Ping{}).Get()
```

Comment:
with this ping, do we even need the rest? since it essentially guarantees container prestart has run before we forward anything to it? then docker would also never receive the signal before it had an id? (could still ping the docker actor for good measure)

Reply:
if we get

reattach{cid: 1} -> cm
signal{cid: 1} -> cm
cm receive reattach
create cid1
ping cid1
  cid1 prestart
  create docker
  ping docker
    docker prestart
    docker reattaches container, sends container reattach
    cid1 inbox container reattach, will always be handled before is forwarded [*]
  docker ack ping
  cid1 -> docker reattach{cid: 1}, will always be handled before is forwarded [*]
cid1 ack ping
cm -> cid1 signal [*], always handled after the last ping
  cid1 is ok

idk, think this is right

Reply:
this description looks like what this code is doing now. which part of this PR are you proposing to get rid of?

Reply:
creating the docker client with a cid is not in this flow, and i think it is still correct

Reply:
but also fine with it now